src/integrationTest/kotlin/com/sourcegraph/cody/edit/DocumentCodeTest.kt

```diff
@@ -37,28 +40,45 @@ class DocumentCodeTest : CodyIntegrationTextFixture() {
         selection.startOffset == caret && selection.endOffset == caret)
   }
 
-  fun skip_testGetsWorkingGroupLens() {
+  fun testGetsWorkingGroupLens() {
+    var lenses: LensWidgetGroup? = null
+    var widgets: MutableList<LensWidget>? = null
+    val showWorkingGroupSessionStateListener =
+        object : FixupService.ActiveFixupSessionStateListener {
+          override fun fixupSessionStateChanged(isInProgress: Boolean) {
+            if (isInProgress) {
+              TestCase.assertNull("Value already set", lenses)
```

Comment:
Assert messages could be more descriptive so if it fails it is immediately obious why